src/util/usfm_to_json.js

```diff
@@ -118,11 +118,13 @@ module.exports = {
                 // Do nothing here for now.
             } else if (splitLine[0].match(new RegExp(/\\m$/gm))) {
                 let cleanedStr = replaceMarkers(line);
-                cleanedStr = "\n" + cleanedStr;
-                book.chapters[c - 1].verses[vnum - 1].verse += ((cleanedStr.length === 0 ? '' : ' ') + cleanedStr);
+                cleanedStr = "\n" + cleanedStr
+                let verseIndex = book.chapters[c - 1].verses.findIndex(val => val.verse_number === vnum);
+                if(book.chapters[c - 1].verses[verseIndex].verse !== undefined)
+                book.chapters[c - 1].verses[verseIndex].verse += ((cleanedStr.length === 0 ? '' : ' ') + cleanedStr);
```

Comment:
Let's add parenthesis after the if to make the code more readable even if it is only one line inside it.

Reply:
okay